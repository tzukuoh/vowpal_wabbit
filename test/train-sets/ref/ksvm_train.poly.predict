0
0.473179
0.043690
-0.236562
-0.379677
-0.449053
-0.161978
-0.386581
-0.290293
-0.501751
-0.497623
-0.410513
-0.524111
-0.490182
-0.530158
-0.335007
-0.205961
-0.052715
-0.371404
-0.381716
-0.372464
-0.088997
-0.491311
-0.029767
0.062841
-0.222215
-0.445744
-0.347848
-0.267142
-0.248318
-0.446773
-0.257801
-0.175833
-0.273587
-0.307128
-0.422144
-0.136477
-0.310648
-0.210800
-0.056006
-0.051921
-0.172945
-0.359546
-0.195782
-0.444699
-0.402708
-0.049628
-0.266273
-0.400910
-0.218478
-0.199653
-0.054293
-0.345500
-0.354551
-0.365320
0.095676
-0.358533
-0.333458
-0.187880
-0.196178
-0.126775
-0.363101
0.084173
-0.630951
-0.300065
-0.365252
-0.345922
-0.115213
-0.436915
-0.103010
-0.096687
0.168128
-0.094265
-0.364062
0.114718
-0.179982
-0.037324
-0.324774
-0.207693
-0.254455
-0.382009
-0.324587
0.007249
-0.244303
-0.468647
-0.404450
-0.425144
-0.102897
-0.162652
-0.707566
-0.257972
-0.174841
0.216237
-0.307258
0.161224
-0.486617
-0.307280
-0.154533
-0.226319
-0.416591
-0.094032
-0.116466
-0.099278
-0.002447
-0.164118
-0.005439
-0.402613
-0.238414
-0.139233
-0.007372
-0.008741
-0.055104
-0.279061
-0.498575
0.352116
-0.098900
-0.316828
-0.261688
-0.050164
-0.302687
-0.053926
0.026317
-0.248577
-0.136705
0.107425
0.191727
-0.068494
0.264938
-0.602269
-0.290196
-0.261435
0.029897
-0.086616
-0.217656
-0.372313
-0.537364
-0.090271
0.097818
0.560413
-0.100463
-0.327482
-0.412552
0.691028
0.724878
-0.339716
0.183549
-0.084537
0.789837
-0.696044
0.061885
0.150081
0.176370
0.397738
-0.168440
-0.483858
0.098324
0.472131
-0.181379
0.660590
-0.068653
0.681352
-0.458187
-0.331585
0.040837
-0.112247
-0.124355
0.507276
-0.240302
-0.363629
0.589560
0.605326
-0.568194
-0.240019
0.104612
-0.676260
0.030655
-0.633424
0.849047
0.216747
0.563965
0.019185
-0.056847
0.257905
-0.423452
-0.117643
-0.104315
0.384207
-0.401813
-0.220614
0.435093
0.470211
-0.038973
-0.098762
-0.360203
-0.296009
-0.023955
0.046168
-0.391004
0.123769
0.088865
-0.293112
-0.041480
0.107085
-0.197253
-0.439831
-0.543661
0.644520
-0.000146
-0.198253
0.096830
0.016185
-0.511201
-0.444373
0.227138
1.101678
-0.716241
-0.356457
-0.305105
-0.168209
-0.173226
0.187826
-0.387302
-0.471680
-0.333836
0.235162
-0.273194
1.138244
0.365616
-0.495638
-0.432104
-0.742711
-0.668235
-0.729619
0.092112
0.313529
-0.646754
-0.470009
0.280130
0.817563
0.318093
0.515428
-0.232987
-0.462125
0.699695
-0.216796
0.145517
0.394339
-0.278859
-0.559065
0.826972
