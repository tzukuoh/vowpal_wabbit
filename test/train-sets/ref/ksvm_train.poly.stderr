using l2 regularization = 1
predictions = ksvm_train.poly.predict
Lambda = 1
Kernel = poly
degree = 2
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.236589 1.473179            2            2.0  -1.0000   0.4732      103
1.070077 0.903564            4            4.0  -1.0000  -0.2366      134
0.975140 0.880204            8            8.0  -1.0000  -0.3866      145
0.934707 0.894274           16           16.0   1.0000  -0.3350       23
0.925318 0.915930           32           32.0  -1.0000  -0.2578       31
0.907599 0.889880           64           64.0  -1.0000  -0.6310       60
0.910526 0.913452          128          128.0   1.0000   0.2649      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.815429
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 247
Number of kernel evaluations = 170641 Number of cache queries = 119344
Total loss = 203.857254
Done freeing model
Done freeing kernel params
Done with finish 
